File: ch1/c/code1_4.c
#include <stdio.h>

int sum(int n)
{
	int sum = 0;
	for(int i = 1; i <= n; i++)
		sum += i;
	return sum;
}

int main()
{  
  printf("숫자를 하나 입력하세요 :");
  int x;
  scanf("%d", &x);
  printf("1에서 %d까지의 합 : %d\n", x, sum(x));
  return 0;
}

File: ch1/python/code1_4.py
def sum(n):
    sum = 0
    for i in range(1, n + 1):
        sum += i
    return sum

n = int(input('숫자를 하나 입력하세요 : '))
print('1에서 %d까지의 합 : %d' % (n, sum(n)))